charts/jupyter/values.yaml

```diff
@@ -73,6 +73,7 @@ jupyterhub:
     defaultUrl: "/lab"
     extraEnv:
       TARGET_URL:
+      ANALYSIS_EXPORT_TARGET:
```

Comment:
my comment is about line 82, the "limit: 1G". During last tests we saw the pods were constantly using 100% memory, and very slow. I'd like to increase it, you agree?

Reply:
this is the limit per user. It can be costly with the number of users we'll soon have. 
I heard that the problem occurred, when there was additional plugin installed (not suitable for fns). 
Let's discuss during the next meeting what would be the reasonable default and if we should double-check the financing of this.